test/mason/publish-tests/publish.good

```diff
@@ -1,2 +1,2 @@
 Created new library project: publishCheck
-Must have remote origin set up in repository to publish
+publish is not a valid path
```

Comment:
The previous error message seems clearer, but I may be misunderstanding the change involved here.